package.json

```diff
@@ -52,6 +52,7 @@
     "@typescript-eslint/eslint-plugin": "^6.7.3",
     "@typescript-eslint/parser": "^6.7.3",
     "@vitest/browser": "^1.4.0",
+    "babel-cli": "^6.26.0",
```

Comment:
Most of the diff is from this. My local laptop was not able to build locally without npm re-installing this. Will remove soon, as I'm assuming it works in CI